students/HABTAMU/session02/fibonacci_serios.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+def fibonacci(n):
+    a, b = 0, 1
+
+    if n <= 0:
+        return 1
+    else:
+        for i in range(n):
+            print(a, end=" ")
+
+            c = a + b
+            a = b
+            b = c
+fibonacci(10)
```

Comment:
Good start.  You can get rid of the blank rows to make this function more compact.  Once this PR is merged, go ahead and do another PR when this assignment is complete.